packages/mui-material/src/Tabs/Tabs.test.js

```diff
@@ -71,6 +71,20 @@ describe('<Tabs />', () => {
     refInstanceof: window.HTMLDivElement,
     testComponentPropWith: 'header',
     testStateOverrides: { prop: 'orientation', value: 'vertical', styleKey: 'vertical' },
+    slots: {
+      root: {
+        expectedClassName: classes.root,
+      },
+      scroller: {
+        expectedClassName: classes.scroller,
+      },
+      flexContainer: {
+        expectedClassName: classes.flexContainer,
+      },
+      indicator: {
+        expectedClassName: classes.indicator,
+      },
```

Comment:
i think tests for `scrollButton`, `startScrollButtonIcon`, `endScrollButtonIcon` are missing

Reply:
Added at the end of this file. I could not leverage `describeConformance` because there are 2 `scrollButtons` when rendered, so the tests will fail due to `queryByTestId`.